### DISCUSSION TITLE
Como é trabalhar sendo um desenvolvedor PJ?

### DISCUSSION THREAD

**Discussion by GuilhermeLis** [frontendbr/forum#1901] [Discussão]
Eu estou começando minha carreira como desenvolvedor PJ agora, mas eu estou bem inseguro com algumas coisas:

- O fato de eu ter que pagar um contador mensalmente, o mais barato que eu encontrei foi a R$ 450,00, caso eu abra o Simples Nacional.

- Como eu vou cuidar de minhas finanças como seguro desemprego, aposentadoria, plano de saúde.

Eu achei o custo do contador muito alto e eu andei pesquisando sobre o MEI e não tem a modalidade de analista de sistemas, não achei alguma modalidade que eu me enquadre então eu estou na dúvida do que fazer.

Será que vale apena? Será que eu estou sendo enrolado? Como viver sendo PJ?

esse é meu primeiro emprego como desenvolvedor PJ eu tenho zero referências.

**Comment by mauriciogallis** [comment]
Boa tarde Guilherme, virei PJ em junho do ano passado e não me arrependo. Mas no meu caso presto serviço para empresa que trabalhava como CLT anteriormente, só emito uma nota por mês e consegui um contador por 180,00 reais por mês. A MEI tem limite de 6,700 reais de nota por mês se não me engano, e o CNAE de analista e programador não se enquadra na MEI por ser uma atividade que possui cunho intelectual, tem muita empresa praticando esse tipo de contratação e aceitando nota de MEI com o CNAE de técnico de informática porém essa prática é ilegal. Aqui onde eu moro conheço diversos casos que a prefeitura desenquadrou algumas pessoas do MEI justamente devido a essa prática. No caso do simples o percentual de imposto para Analista/Programador é de 14% porém você consegue se enquadrar no anexo III se tirar um pró-labore de 28%(esses 28% você vai ter que recolher INSS/IRRF) e conseguir pagar  6% de simples que é a forma que eu trabalho. No final das contas tudo vai depender da proposta CLTxPJ.

**Comment by GuilhermeLis** [comment]
@mauriciogallis eu andei pesquisando, mas ainda não sei dizer em qual situação ser PJ se torna mais vantajoso que ser CLT

**Comment by giovannism20** [comment]
@GuilhermeLis Não sei se vai exatamente te ajudar mas, existem casos sim. Neste [vídeo](https://youtu.be/6fVmOs9lcok?t=265) presente no canal da DevMedia eles fazem a comparação e contas iniciais que você faria para comparar o quanto vale e/ou se vale a pena escolher entre a vaga CLT - PJ.

**Comment by mauriciogallis** [comment]
@GuilhermeLis a grande questão é analisar a proposta, mas pra ser bem resumido, uma vaga PJ tem que ter um salário mensal em média entre 35 e 40% maior do que a mesma vaga no CLT.

**Comment by jancassio** [comment]
Eu sou PJ há pelo menos 11 anos mas ja faz um tempo que eu não conseguia me ver mais como um dev contratado carteira, pq PJ tem mais liquidez e de certa forma, os encargos não são os mesmos de CLT.

No começo essa insegurança é normal. Com o tempo, vc vai aprender a colocar despesas e encargos na ponta do lápis e conseguir calibrar a liquidez que vc merece e precisa para o seu caso.

> O fato de eu ter que pagar um contador mensalmente, o mais barato que eu encontrei foi a R$ 450,00, caso eu abra o Simples Nacional.

Você só precisa pagar contador se vc não quiser ou não conseguir fazer a parte de contabilidade da sua empresa. Eu pago pq simplesmente não tenho cabeça pra essas coisas.

Não sei se posso recomendar aqui (me desculpem) mas o meu é uma chamada [Ibrage](https://www.ibrage.com.br/). Fale com o Victor e ele vai te ajudar no que precisar. Se quiser, fala que foi recomendação minha (não vou ganhar nada com isso mas sei lá haha).

> Como eu vou cuidar de minhas finanças como seguro desemprego, aposentadoria, plano de saúde.

Simples, vc não tem essas garantias, por isso mesmo é colocando as contas na ponta do lápis que vai te ajudar a ter o retorno suficiente pra que vc possa se planejar e controlar melhor suas finanças.

Algumas regras que posso compartilhar que uso e podem te ajudar:

* Projetos com escopo muito aberto - não feche um fee mensal, negocie as horas trabalhadas e se possível, defina no acordo que vc vai ter um piso mínimo de horas - O valor hora neste caso **deve** ser maior que o valor hora q vc está acostumado a cobrar.

* Projetos longos - geralmente valem a pena fechar um fee com um acordo de horas extras mas na minha experiência, poucas vezes precisei add horas. Cobre bem neste caso!

* Projetos curtos com escopo fechado - Fecha um valor p/ o projeto se vc sentir q ele ta redondo e q o cronograma é tangível, se ele dor durar mais q 3 meses, talvez seja melhor fechar por fee mesmo pq projetos de longo prazo, mudam o escopo com mais frequencia.

De resto, seja *muito* profissional, pq isso vai te dar mais retorno a longo prazo do que a grana. As empresas e pessoas costumam ir atrás de pessoas que eles ja tiveram um bom relacionamento anterior, isso se propaga no mercado e vc faz um nome. Então **valorize seu nome no mercado** e vc sempre ficará bem.

**Comment by dionarodrigues** [comment]
@GuilhermeLis O seu contador está cobrando um valor muito alto. Eu utilizo a [Contabilizei](https://www.contabilizei.com.br/), assim como muitos devs, e pago apenas 90,00/mês. É tudo digital e tem canais pra vocês entrar em contato com eles via chat ou chamado - sempre que precisei deles, foi fácil fazer a comunicação. Antes disso, eu pagava 350,00 de contador e não era tão visível o que eu estava pagando e porque. 😒 Na Contabilizei tudo é muito visível.  Existem outras plataformas digitais também que você pode pesquisar. 

Sobre cuidar das suas finanças, geralmente o contador te auxilia, mas basicamente será:
- No final do mês, somar o valor de todas as notas que você emitiu
- Definir o seu pró-labore (que geralmente é 28% em cima do total das notas)
- Aguardar as guias para pagamento: DAS (Simples Nacional - geralmente 6% em cima do total das notas), IRRP (Imposto de Renda) e INSS (que será 11% em cima do valor do pró-labore).

Tem um [thread aqui sobre isso](https://github.com/frontendbr/forum/issues/471).

É importante que na hora de dizer o quanto você deseja ganhar como PJ, você inclua o valor referente ao 13º, férias e outros benefícios. Então vc soma o valor líquido que vc deseja + benefícios + gastos que terá por mês. Não me lembro agora, mas o contratante, no modelo CLT, chega a gastar quase o dobro a mais com o funcionário, se eu não me engano. Então, você teria que encontrar um meio termo para que os dois saiam ganhando no modelo PJ. Também é preciso pesquisar, mas se eu não me engano, no PJ o valor precisa ser em torno de 70% a mais do que você ganharia como CLT para valer a pena. 

Na internet tem várias calculadoras de comparativos entre CLT x PJ para você analisar.

---

Sobre ser PJ ao invés de CLT, para mim é uma delicinha.  😁 Me sinto muito mais livre e no controle dos meus impostos. Sabe aquele momento, no modelo CLT, em que não queremos mais trabalhar em alguma empresa ou quando recebemos uma oportunidade melhor e não queremos pedir demissão pra não deixar de pegar o fundo de garantia? Então, isso não existe no modelo PJ. 😁 Você não recebe nada, por isso é importante incluir os seus benefícios no salário e guardar o valor referente as férias e 13º, além do que você contribuiria para o fundo de garantia. Então, perceba que o pulo do gato aqui é conseguir se organizar financeiramente, fazendo uma boa gestão do seu salário - e não achar que tá rico porque tá ganhando mais do que ganharia como CLT e sair gastando tudo, porque depois pode fazer muuuita falta rsss. Mas depois você pega o jeito e tudo flui naturalmente.

Um detalhe bem importante para se discutir na contração como PJ são as férias. PJ não é igual CLT que possui férias por lei. Então, é interessante negociar. Tem empresas que dá férias remuneradas, tem outras que não remunera (aí você precisa se organizar financeiramente). Tem as que dão 30 dias, tem as que dão só no final do ano...  E tem as que não dão hahaha

Bom, essas são minhas opiniões. 
Espero ter ajudado. Boa sorte!

**Comment by GuilhermeLis** [comment]
@diogorodrigues obrigado amigo... ajudou muito

**Comment by lucascordeiroaraujo** [comment]
@GuilhermeLis Trabalho a 1 ano como PJ, vou compartilhar contigo como faço.

Assim como o @diogorodrigues eu utilizo a [Contabilizei](https://www.contabilizei.com.br/programa-de-indicacao?ref=57a0bb7f1a20557b0d6fb062808602d3&utm_source=plataforma&utm_campaign=MGM&utm_source=plataforma) pago **R$ 89,90** por mês;

A empresa para qual trabalho tem sede em São Paulo e eu moro em Florianópolis, eu resolvi realizar a abertura do meu CNPJ em São Paulo mesmo, para facilitar e não ter a retenção de ISS, que pode variar de 2% a 5%. A abertura da ME fiz na [Contabilizei](https://www.contabilizei.com.br/programa-de-indicacao?ref=57a0bb7f1a20557b0d6fb062808602d3&utm_source=plataforma&utm_campaign=MGM&utm_source=plataforma) também, não me recordo qual a valor que eles cobram.

Com essa decisão de abrir a empresa em SP eu precisava de um endereço fiscal de lá, para isso utilizei o serviço da [Company Hero](https://www.companyhero.com/) onde pago **R$ 1.176,00** por ano.

Tem também o Certificado Digital A1 que você tem que renovar todo ano, no valor de **R$ 175,00**

A cada nota que eu emito o percentual de imposto é 6% na atividade `Desenvolvimento de programas de computador sob encomenda`

É um pouco burocrática a abertura / manutenção quanto se está como PJ, mas vale a pena 😄 

Agora tem também a MEI ai vai depender se você vai emitir mais que **R$ 6.750,00** por mês. Optando pelo MEI você paga menos impostos e é mais fácil a abertura / manutenção mensal. Não precisa nem de contador.

Abraço.

**Comment by jonasinfomei** [comment]
Olá pessoal, o tópico é antigo, mas muitas coisas mudaram de lá pra cá, e com a contabilidade digital, a contabilidade para devs e programadores melhorou muito e ficou mais acessível, eu tenho um conteúdo que auxilia nas duvidas para passar de um desenvolvedor freelancer para PJ, caso tenham interesse só [clicar aqui](https://www.ngconsultoriacontabil.com/desenvolvedor-pj-como-abrir-um-cnpj-para-desenvolvedor/) ok.

Obrigado.
